lib/process-services/app-list/icon.model.ts

```diff
@@ -142,7 +142,7 @@ export class IconModel {
         this.iconsMDL.set('glyphicon-gbp', IconModel.DEFAULT_TASKS_APP_MATERIAL_ICON);
         this.iconsMDL.set('glyphicon-sort', IconModel.DEFAULT_TASKS_APP_MATERIAL_ICON);
         this.iconsMDL.set('glyphicon-flash', IconModel.DEFAULT_TASKS_APP_MATERIAL_ICON);
-        this.iconsMDL.set('glyphicon-record', IconModel.DEFAULT_TASKS_APP_MATERIAL_ICON);
+        this.iconsMDL.set('glyphicon-record', 'radio_button_checked');
```

Comment:
all other icons are using IconModel. Should be consistent

Reply:
The icon model is used in case we are not able to find the correct mapping between glyphicon and material icon